pkgs/development/libraries/git2/default.nix

```diff
@@ -1,18 +1,17 @@
-{ stdenv, fetchFromGitHub, pkgconfig, cmake
-, zlib, python, libssh2, openssl, curl, http-parser
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, python
+, zlib, libssh2, openssl, http-parser, curl
 , libiconv, Security
 }:
 
-stdenv.mkDerivation (rec {
+stdenv.mkDerivation rec {
+  version = "0.27.8";
   name = "libgit2-${version}";
-  version = "0.26.6";
-  # keep the version in sync with pythonPackages.pygit2 and libgit2-glib
```

Comment:
Why remove this comment?

Reply:
I did `git rm default.nix` and `git mv 0.27.nix default.nix` but didn't notice that comment was only present in `default.nix`. I'll add that again. Thanks!